scripts/generator/graphql_generator/csharp/SDK/CheckoutUtils.cs.erb

```diff
@@ -0,0 +1,22 @@
+namespace <%= namespace %>.SDK {
+    using System.Collections.Generic;
+    using <%= namespace %>.MiniJSON;
+    
+    public class CheckoutUtils {
```

Comment:
I don't think we should be abstracting this out of `iOSNativeCheckout`, reason being is https://github.com/Shopify/unity-buy-sdk/issues/213 where we need to change `TOTAL` to shop name. 

Since both platforms are expecting different formats of `SummaryItem` I think we should have a subclass of `SummaryItem` for each platforms that returns a `SerializedSummaryItem` in the format that each platform uses. A list for iOS and a dictionary for Android